# YOUR DATASET PATH
dataset_path: /home/yw/vance/git/catkin_ws/data/

camera_intrinsic: [
  968.31466,0,813.0669,
  0,965.046,287.335,
  0,0,1
]

camera_extrinsic: [
  -0.000332209, -0.999917, -0.0128807, -0.025611,
  0.00470448, 0.012879, -0.999906, 0.0377289,
  0.999989, -0.000392775, 0.00469981, 0.137318,
  0, 0, 0, 1
]

lidar_lidar_extrinsic: [
  0,	0,	0,	0,	0,	0,
  0.205121,	0.0335168,	0.0895346,	-0.00760184,	0.0459168,	-0.105951,
  0.261006,	-0.0978569,	0.0540669,	-0.0031703,	-0.00298965,	-0.0901403,
  0.0116515,	0.103358,	0.229466,	0.0069734,	-0.0912874,	-0.108668,
  0.29819,	0.183213,	0.1946,	-0.0116543,	0.0541184,	-0.00368836,
  -179.746,	179.952,	179.641,	-0.00704259,	-0.0944232,	0.0163535
]

lidar_to_apx_extrinsic: [
  -0.999695, -0.0244322, 0.00348961,   -1.2,
  0.024432, -0.999701,  -8.52842e-05, -0.2,
  0.00349065,0.0,       0.999994,     0,
  0,0,0,1
]

camera_factor: 256.0
imageRows: 568
imageCols: 1520 

sparse_optical_flow_param:
  maxCorners: 5000
  qualityLevel: 0.0001
  minDistance: 5
  blockSize: 3
  Harris_k_value: 0.05